blog/index.html

```diff
@@ -23,11 +23,41 @@ <h2>{{ site.blog_description }}</h2>
 
   <ul class="post-list">
     {% for post in paginator.posts %}
-      <li>
-        <h3><a class="post-title" href="{{ post.url | prepend: site.baseurl }}">{{ post.title }}</a></h3>
-        <p class="post-meta">{{ post.date | date: '%B %-d, %Y' }}</p>
-        <p>{{ post.description }}</p>
-      </li>
+
+    {% assign read_time = page.content | number_of_words | divided_by: 180 | plus: 1 %}
```

Comment:
really like the read time feature!

```suggestion
    {% assign read_time = page.content | number_of_words: "auto" | divided_by: 180 | plus: 1 %}
```